data/releases/5.2.0.md

```diff
@@ -8,12 +8,12 @@ intro: |
   2024-05-13. Go [here](/releases) for a list of all releases.
 
   This release is available as an [opam](/p/ocaml/5.2.0) package.
-highlights: |
-  - Re-introduced GC compaction
+Hhighlights: |
```

Comment:
```suggestion
highlights: |
```

This is part of the YAML structure, and not part of the text. Changing the capitalization on this breaks the build.

Reply:
Oh! Apologies (and a typo to boot!)